src/main/app/src/app/zaken/zaak-verkort/zaak-verkort.component.ts

```diff
@@ -15,9 +15,9 @@ import { Zaak } from "../model/zaak";
   styleUrls: ["./zaak-verkort.component.less"],
 })
 export class ZaakVerkortComponent implements OnChanges {
-  @Input() zaak: Zaak;
+  @Input() zaak!: Zaak;
```

Comment:
if we know it is passed for sure we should mark it as required

```suggestion
  @Input({ required: true }) zaak!: Zaak;
```